### PR TITLE
JinJang: Prepare for WP.org submission

### PR DESCRIPTION
**Pull request by iamtakashi**
#### Changes proposed in this Pull Request:
There are minor improvements and cleanup to prepare for WP.org submission. My apologies for the amount of changes! 

* Changed the name casing `jinjang` > `JinJang` (**Would this cause a problem in WP.com?**)
* Cleaned up the CSS header and readme.
* Corrected the translation functions in 404.
* Added comment template part, and removed the pattern for it.
* Added comment section to the single post template. 
* Improved markup for all the templates.
* Swapped the positions of the post title and the post date.
* Increased the spacing between the posts in index, archive, and search templates.
* Cleaned up the CSS.
* Various improvements in block stylings.
* A new screenshot reflects the changes made.

### CODE REVIEW

File: jinjang/templates/page.html [hunk 1]
```diff
@@ -14,20 +14,20 @@
...
 					<!-- wp:post-title /-->
 
 					<!-- wp:post-content {"lock":{"move":false,"remove":false},"layout":{"type":"constrained","justifyContent":"center"}} /-->
-				</main>
+				</div>
 				<!-- /wp:group -->
 
-				<!-- wp:pattern {"slug":"jinjang/comments"} /-->
-			</div>
+				<!-- wp:template-part {"slug":"comments"} /-->
```
Comment (vcanales) [inline jinjang/templates/page.html:29]: The comments template part needs to be included.

image

File: jinjang/templates/single.html [hunk 2]
```diff
@@ -45,11 +45,13 @@
 					<!-- wp:post-navigation-link /-->
 				</div>
 				<!-- /wp:group -->
-			</div>
+
+				<!-- wp:template-part {"slug":"comments"} /-->
```
Comment (vcanales) [inline jinjang/templates/single.html:49]: Same as above; it seems like the comments template part file is missing.

**Review by vcanales — CHANGES_REQUESTED** [review]

File: jinjang/templates/page.html (hunk 1 above)
Comment (iamtakashi) [reply jinjang/templates/page.html:29]: Sorry, what am I missing? I can't replicate the missing comment template on `page.html` and `single.html`. I tested it with/without Gutenberg, but it worked for me.

Screenshot 2023-10-13 at 09 32 06

**Comment by iamtakashi** [comment]
@vcanales Have you had a chance to look at why we see different things? Any advice would be appreciated.

**Comment by vcanales** [comment]
> @vcanales Have you had a chance to look at why we see different things? Any advice would be appreciated.

The changes on GitHub show that the comments.php file has been deleted, but it doesn't show a comments template part being added; could you export the theme again, create a new zip, and confirm that the comments template file is there?

**Comment by iamtakashi** [comment]
I've likely forgotten to add the comment template part in the initial commit. It should be good now. Could you review it again? Thanks, @vcanales!

**Comment by iamtakashi** [comment]
Hi @vcanales, would you be able to re-review this one anytime soon? We want to keep feeding the theme review team as they only take one theme from an account at a time. Please let me know if there is anything I can to move this forward. Thanks in advance!

**Review by vcanales — APPROVED** [review]
All good now! Merging, and deploying to dotcom as well.

**Comment by iamtakashi** [comment]
Thank you @vcanales!!

**Pull request merged**